WooCommerce/Classes/POS/Controllers/PointOfSaleOrderController.swift

```diff
@@ -12,6 +12,7 @@ protocol PointOfSaleOrderControllerProtocol {
     var order: Order? { get }
 
     func syncOrder(for cartProducts: [CartItem], retryHandler: @escaping () async -> Void) async
+    func sendOrderReceipt(order: Order, toEmailAddress: String) async
```

Comment:
Thanks! This is the same case as https://github.com/woocommerce/woocommerce-ios/pull/14481#discussion_r1866978975, if I don't add the method to the protocol here, then we need another way to call it through the `POSAggregateModel` as we need access through the interface. My understanding is that we'd use the controller to interact with these? What would be preferable here rather than using the order controller?
- Pass the OrderService to the POSAggregateModel?
- Pass a new receiptsController protocol? (this approach seems to make more sense, as we already have items controller, and order controller)